docs/html/user_guide.rst

```diff
@@ -94,10 +94,11 @@ pip can be configured to connect through a proxy server in various ways:
 * using the ``--proxy`` command-line option to specify a proxy in the form
   ``scheme://[user:passwd@]proxy.server:port``
```

Comment:
If there's any need for explicitly calling out `PIP_PROXY` (note that uppercase is the canonical form here) I'd much rather it was by linking to the general docs on using environment variables to configure command line options [here](https://pip.pypa.io/en/stable/topics/configuration/#environment-variables).

I guess the fact that we already document config files independently sort of contradicts this.

Maybe we should have something like

```
* using the  ``--proxy`` command-line option to specify a proxy in the form
  ``scheme://[user:passwd@]proxy.server:port``. For persistent configuration,
  the standard approaches of using a :ref:`config-file` or :ref:`environment-variable`
  are available.
```